modules/cluster-logging-release-notes-5.0.0.adoc

```diff
@@ -19,23 +19,23 @@ With this release, Cluster Logging becomes Red Hat OpenShift Logging, version 5.
 // https://bugzilla.redhat.com/show_bug.cgi?id=1883444
 === Maximum five primary shards per index
 
-With this release, the Elasticsearch Operator (EO) sets the number of primary shards for an index between one and five, depending on the number of data nodes defined for a cluster.
+With this release, the OpenShift Elasticsearch Operator (EO) sets the number of primary shards for an index between one and five, depending on the number of data nodes defined for a cluster.
 
 Previously, the EO set the number of shards for an index to the number of data nodes. When an index in Elasticsearch was configured with a number of replicas, it created that many replicas for each primary shard, not per index. Therefore, as the index sharded, a greater number of replica shards existed in the cluster, which created a lot of overhead for the cluster to replicate and keep in sync.
 
 [discrete]
 [id="openshift-logging-5-0-updated-eo-name"]
 // https://bugzilla.redhat.com/show_bug.cgi?id=1898920
-=== Updated Elasticsearch Operator name and maturity level
+=== Updated OpenShift Elasticsearch Operator name and maturity level
 
-This release updates the display name of the Elasticsearch Operator and operator maturity level. The new display name and clarified specific use for the Elasticsearch Operator are updated in Operator Hub.
+This release updates the display name of the OpenShift Elasticsearch Operator and operator maturity level. The new display name and clarified specific use for the OpenShift Elasticsearch Operator are updated in Operator Hub.
 
 [discrete]
 [id="openshift-logging-5-0-es-csv-success"]
 // https://bugzilla.redhat.com/show_bug.cgi?id=1913464
-=== Elasticsearch Operator reports on CSV success
+=== OpenShift Elasticsearch Operator reports on CSV success
 
-This release adds reporting metrics to indicate that installing or upgrading the Elasticsearch Operator ClusterServiceVersion (CSV) was successful. Previously, there was no way to determine, or generate an alert, if the CSV installation or upgrade for the Elasticsearch Operator failed. Now, an alert is provided as part of the Elasticsearch Operator.
+This release adds reporting metrics to indicate that installing or upgrading the OpenShift Elasticsearch Operator ClusterServiceVersion (CSV) was successful. Previously, there was no way to determine, or generate an alert, if the CSV installation or upgrade for the OpenShift Elasticsearch Operator failed. Now, an alert is provided as part of the OpenShift Elasticsearch Operator.
```

Comment:
Probably out of scope for this, but just noting that ClusterServiceVersion is a custom resource and should probably be written as "`ClusterServiceVersion` object (CSV)" or "`ClusterServiceVersion` custom resource (CSV)" according to standards. I see this one is used very inconsistently in the repo, though.

Reply:
Thanks. Updated.